roles/ocluster/tasks/main.yml

```diff
@@ -23,15 +23,15 @@
 
 - name: Download the source from GitHub
   git:
-    repo: https://github.com/mtelvers/ocluster.git
+    repo: https://github.com/ocurrent/ocluster.git
     dest: "{{ ansible_env.HOME }}/ocluster"
     force: yes
     recursive: no
     version: freebsd
```

Comment:
```suggestion

```

`version:` needs to be removed from both plays as this is the name of the branch.

Reply:
I removed the `version` line and re-pushed the branch.